console/area.geojson

```diff
@@ -0,0 +1,365 @@
+{ "type": "Feature", "properties": { "ID": "COMMUNE_0000000009763508", "NOM_COM_M": "TAVACO", "INSEE_COM": "2A323", "INSEE_CAN": "08", "INSEE_ARR": "1", "INSEE_DEP": "2A" }, "geometry": { "type": "Polygon", "coordinates": [ [ [ 8.87, 42.049 ], [ 8.869, 42.044 ], [ 8.87, 42.043 ], [ 8.87, 42.04 ], [ 8.867, 42.037 ], [ 8.866, 42.034 ], [ 8.866, 42.032 ], [ 8.865, 42.031 ], [ 8.867, 42.026 ], [ 8.87, 42.024 ], [ 8.869, 42.022 ], [ 8.87, 42.02 ], [ 8.869, 42.018 ], [ 8.868, 42.016 ], [ 8.867, 42.015 ], [ 8.869, 42.015 ], [ 8.871, 42.012 ], [ 8.876, 42.015 ], [ 8.882, 42.014 ], [ 8.884, 42.01 ], [ 8.888, 42.012 ], [ 8.889, 42.015 ], [ 8.892, 42.016 ], [ 8.893, 42.017 ], [ 8.897, 42.02 ], [ 8.898, 42.021 ], [ 8.9, 42.021 ], [ 8.903, 42.023 ], [ 8.905, 42.026 ], [ 8.907, 42.026 ], [ 8.909, 42.03 ], [ 8.908, 42.031 ], [ 8.91, 42.036 ], [ 8.908, 42.039 ], [ 8.905, 42.039 ], [ 8.904, 42.04 ], [ 8.902, 42.041 ], [ 8.901, 42.039 ], [ 8.897, 42.04 ], [ 8.898, 42.041 ], [ 8.897, 42.042 ], [ 8.895, 42.043 ], [ 8.895, 42.044 ], [ 8.894, 42.045 ], [ 8.89, 42.046 ], [ 8.887, 42.048 ], [ 8.882, 42.05 ], [ 8.881, 42.052 ], [ 8.877, 42.056 ], [ 8.876, 42.054 ], [ 8.87, 42.049 ] ] ] } },
+{ "type": "Feature", "properties": { "ID": "COMMUNE_0000000009763028", "NOM_COM_M": "LAVATOGGIO", "INSEE_COM": "2B138", "INSEE_CAN": "07", "INSEE_ARR": "5", "INSEE_DEP": "2B" }, "geometry": { "type": "Polygon", "coordinates": [ [ [ 8.877, 42.561 ], [ 8.877, 42.562 ], [ 8.875, 42.565 ], [ 8.879, 42.568 ], [ 8.88, 42.568 ], [ 8.883, 42.57 ], [ 8.885, 42.572 ], [ 8.884, 42.576 ], [ 8.881, 42.576 ], [ 8.88, 42.578 ], [ 8.877, 42.577 ], [ 8.873, 42.578 ], [ 8.871, 42.58 ], [ 8.868, 42.582 ], [ 8.865, 42.582 ], [ 8.863, 42.583 ], [ 8.86, 42.583 ],
```

Comment:
I think you can filter out the `ID`, `INSEE_CAN`, `INSEE_ARR` fields which are useless and make the response heavier.